deepmd/pt/cxx_op.py

```
@@ -87,6 +87,35 @@
+    elif platform.system() == "Darwin":
+        lib_env = "DYLD_FALLBACK_LIBRARY_PATH"
+        exition = ".dylib"
+    else:
+        # windows
+        pass
+
+    if platform.system() in ("Linux", "Darwin"):
+        # try to find MPI directory and add to LD_LIBRARY_PATH
+        try:
+            MPI_ROOT = (
+                [p for p in metadata.files("mpich") if f"libmpi{extension}" in str(p)][0]
```

Ruff (0.12.2)

Undefined name `metadata`

(F821)